src/utils.ts

```diff
@@ -16,3 +19,36 @@ export type UnwrapPromise<T> = T extends PromiseLike<infer R> ? UnwrapPromise<R>
 
 // eslint-disable-next-line @typescript-eslint/no-empty-function, @typescript-eslint/no-unused-vars
 export function noop(..._args: unknown[]): void {}
+
+export type InferBrowserPluginArray<
```

Comment:
this is considered art 🤩 
what is the current impact on the language server when unwrapping the array?

Reply:
That... I don't know, I've never actually measured that before 😓 
Any tips on how to do it?

Reply:
That shouldn't have a significant impact imo. Many other packages do unwrapping as well.